Test failures with Python >=3.3

Nose trunk fails some tests with Python >=3.3 due to changed output (added quotes).

```
======================================================================
FAIL: Doctest: test_config_defaults.rst
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python3.3/doctest.py", line 2179, in runTest
    raise self.failureException(self.format_failure(new.getvalue()))
AssertionError: Failed doctest test for test_config_defaults.rst
  File "/tmp/nose/build/tests/unit_tests/test_config_defaults.rst", line 0

----------------------------------------------------------------------
File "/tmp/nose/build/tests/unit_tests/test_config_defaults.rst", line 92, in test_config_defaults.rst
Failed example:
    options, args = parse([], StringIO("spam"))
Expected:
    error: Error reading config file '<???>': File contains no section headers.
    file: <???>, line: 1
    'spam'
Got:
    error: Error reading config file '<???>': File contains no section headers.
    file: '<???>', line: 1
    'spam'
----------------------------------------------------------------------
File "/tmp/nose/build/tests/unit_tests/test_config_defaults.rst", line 99, in test_config_defaults.rst
Failed example:
    options, args = parse([], os.path.join(support, "invalid.cfg"))
    # doctest: +ELLIPSIS
Expected:
    error: Error reading config file '...invalid.cfg': File contains no section headers.
    file: ...invalid.cfg, line: 1
    'spam\n'
Got:
    error: Error reading config file '/tmp/nose/build/tests/unit_tests/support/config_defaults/invalid.cfg': File contains no section headers.
    file: '/tmp/nose/build/tests/unit_tests/support/config_defaults/invalid.cfg', line: 1
    'spam\n'
----------------------------------------------------------------------
File "/tmp/nose/build/tests/unit_tests/test_config_defaults.rst", line 107, in test_config_defaults.rst
Failed example:
    options, args = parse([], [os.path.join(support, "invalid.cfg")])
    # doctest: +ELLIPSIS
Expected:
    error: Error reading config file '...invalid.cfg': File contains no section headers.
    file: ...invalid.cfg, line: 1
    'spam\n'
Got:
    error: Error reading config file '/tmp/nose/build/tests/unit_tests/support/config_defaults/invalid.cfg': File contains no section headers.
    file: '/tmp/nose/build/tests/unit_tests/support/config_defaults/invalid.cfg', line: 1
    'spam\n'
----------------------------------------------------------------------
File "/tmp/nose/build/tests/unit_tests/test_config_defaults.rst", line 118, in test_config_defaults.rst
Failed example:
    options, args = parse([], [os.path.join(support, "a.cfg"),
                               os.path.join(support, "invalid.cfg"),
                               os.path.join(support, "b.cfg")])
    # doctest: +ELLIPSIS
Expected:
    error: Error reading config file '...invalid.cfg': File contains no section headers.
    file: ...invalid.cfg, line: 1
    'spam\n'
Got:
    error: Error reading config file '/tmp/nose/build/tests/unit_tests/support/config_defaults/invalid.cfg': File contains no section headers.
    file: '/tmp/nose/build/tests/unit_tests/support/config_defaults/invalid.cfg', line: 1
    'spam\n'


----------------------------------------------------------------------
Ran 383 tests in 40.153s

FAILED (SKIP=8, failures=1)
```

--- a/unit_tests/test_config_defaults.rst
+++ b/unit_tests/test_config_defaults.rst
@@ -89,9 +89,9 @@ Invalid config files
 
 (file-like object)
 
-    >>> options, args = parse([], StringIO("spam"))
+    >>> options, args = parse([], StringIO("spam")) # doctest: +ELLIPSIS
     error: Error reading config file '<???>': File contains no section headers.
-    file: <???>, line: 1
+    file: ...<???>..., line: 1
     'spam'
 
 (filename)
@@ -99,7 +99,7 @@ Invalid config files
     >>> options, args = parse([], os.path.join(support, "invalid.cfg"))
     ... # doctest: +ELLIPSIS
     error: Error reading config file '...invalid.cfg': File contains no section headers.
-    file: ...invalid.cfg, line: 1
+    file: ...invalid.cfg..., line: 1
     'spam\n'
 
 (filenames, length == 1)
@@ -107,7 +107,7 @@ Invalid config files
     >>> options, args = parse([], [os.path.join(support, "invalid.cfg")])
     ... # doctest: +ELLIPSIS
     error: Error reading config file '...invalid.cfg': File contains no section headers.
-    file: ...invalid.cfg, line: 1
+    file: ...invalid.cfg..., line: 1
     'spam\n'
 
 (filenames, length > 1)
@@ -120,7 +120,7 @@ file is bad
     ...                            os.path.join(support, "b.cfg")])
     ... # doctest: +ELLIPSIS
     error: Error reading config file '...invalid.cfg': File contains no section headers.
-    file: ...invalid.cfg, line: 1
+    file: ...invalid.cfg..., line: 1
     'spam\n'
 
 

